testdata/README.md

```diff
@@ -7,14 +7,18 @@ output is expressed in hexadecimal byte notation.
 
 ## ES6 Numbers
 
-For testing ES6 number serialization there is a ZIP file on 
-https://1drv.ms/u/s!AmhUDQ0Od0GTiXeAjaBJFLJlxyg0?e=HFG4Ao
-containing about a 100 million of random and edge-case values.  The test file consists of lines
+For testing ES6 number serialization there is a GZIP file hosted at
+https://github.com/dsnet/json-canonicalization/releases/download/es6testfile/es6testfile100m.txt.gz
```

Comment:
The test file is just under the 2GiB limit for binaries that can be attached to a release: https://docs.github.com/en/github/managing-large-files/working-with-large-files/distributing-large-binaries

I've temporarily put a mirror from my fork, but ideally this should point to this repo. However, I don't have priviledges to make a release here.

Reply:
The attached with generated using `numgen.go`.